.github/workflows/ci.yml

```diff
@@ -253,13 +254,13 @@ jobs:
     strategy:
       matrix:
         python-version: [3.8]
-        os: ['macos-12', 'windows-2019']
+        os: ['macos-latest', 'windows-latest']
```

Comment:
would it be possible to fix this to whatever the latest currently is?

otherwise the next upgrade can break the test without us knowing why.

Reply:
Done :+1: